hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/SequenceIdAccounting.java

```diff
@@ -163,6 +162,35 @@ void update(byte[] encodedRegionName, Set<byte[]> families, long sequenceid,
     }
   }
 
+  void updateStore(byte[] encodedRegionName, byte[] familyName, Long sequenceId,
```

Comment:
Comments please ... What is the purpose of this method (name very vague?). Why package-visibility?